### DISCUSSION TITLE
Windows Git Clone Error    "error: filename in tree entry contains backslash: '\'.32 MiB/s"

### DISCUSSION THREAD

**Discussion by sloanm** [community/community#23269] [New to GitHub]
<p>Anyone know how to correct for this error?</p>
<p>Thanks</p>
<p>Windows Git Clone Error</p>
<p>PS C:\Users\username1\Documents&gt; git clone <a href="https://github.company.com/Operations/WinProvision" rel="nofollow noopener">https://github.company.com/Operations/WinProvision</a><br>
Cloning into ‘WinProvision’…<br>
remote: Enumerating objects: 152, done.<br>
remote: Counting objects: 100% (152/152), done.<br>
remote: Compressing objects: 100% (110/110), done.<br>
error: filename in tree entry contains backslash: ‘’.32 MiB/s<br>
fatal: Not all child objects of 65c0283643f are reachable<br>
fatal: index-pack failed</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>See the thread here: <a href="https://github.com/civetweb/civetweb/issues/812" rel="nofollow noopener">https://github.com/civetweb/civetweb/issues/812</a> if you update to git 2.25 it should work as expected. </p>

**Comment by sloanm** [comment]
<p>Absent the upgrade for the time being I used</p>
<pre><code>git config --global core.protectNTFS false
</code></pre>
<p>As mentioned in the link and that worked.</p>

**Comment by 0980868366** [comment]
<pre><code>git config --global core.protectNTFS false
</code></pre>

**Comment by 0980868366** [comment]
<p>Hi </p>
<p> there a</p>
<blockquote>
<p><a class="mention" href="/0980868366">@0980868366</a> wrote:</p>
</blockquote>
<blockquote>
<p>git config --global core.protectNTFS false</p>
</blockquote>

**Comment by rexparis** [comment]
<p>github desktop ver 2.40 gives the error and there is no updates available as of today.  Using git bash ver2.26 i did not get the error and was able to clone the repository. </p>
<p>thanks for the solution. </p>

**Comment by PriitTiganik** [comment]
<p>Version 2.4.3 seems to throw the same error</p>
